pkg/util/util.go

```diff
@@ -83,6 +83,14 @@ func IsSEVVMI(vmi *v1.VirtualMachineInstance) bool {
 	return vmi.Spec.Domain.LaunchSecurity != nil && vmi.Spec.Domain.LaunchSecurity.SEV != nil
 }
 
+// Check if a VMI spec requests AMD SEV
+func IsSEVESVMI(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
Can we add the check if the vmi is equal to nil and return false if so?

Reply:
Everything else in the util package assumes the vmi instance is present and valid or is handled in the caller before calling into the util package. I think it's safe to assume the same for SEV and SEV-ES and not add further clutter here.